nixos/modules/programs/fish.nix

```diff
@@ -205,6 +217,11 @@ in
         # if we haven't sourced the interactive config, do it
         status --is-interactive; and not set -q __fish_nixos_interactive_config_sourced
         and begin
+          ${concatStringsSep "\n" (
```

Comment:
Could we make this `${fishAbbrs}` and define `fishAbbrs` above (like `fishAliases` is)?